### PR TITLE
Voice Enabled Coffee Maker powered by Windows 10 IoT Core

### PR DESCRIPTION
**Pull request by trmck**
Using a Raspberry Pi 2, Black and Decker coffee maker and Windows 10 IoT Core I wrote a sample that allows voice commands to turn the maker on and off. There's also a web server running in the background that allows the calling of endpoints to control the maker as well. The plan is to allow the endpoints to be called by Cortona to allow control from a Windows Phone

### CODE REVIEW

File: Raspberry Coffee/CoffeeMakerControllerApp/Package.appxmanifest [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
+  <Identity Name="CoffeeMakerExtendedController" Publisher="CN=trmck" Version="1.0.0.0" />
+  <mp:PhoneIdentity PhoneProductId="834b37a0-3156-4759-84b7-57242a9820d9" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
+  <Properties>
+    <DisplayName>Cortona Coffee Application</DisplayName>
+    <PublisherDisplayName>trmck@microsoft.com</PublisherDisplayName>
```
Comment (bfjelds) [inline Raspberry Coffee/CoffeeMakerControllerApp/Package.appxmanifest]: use MSFT as the publisher

File: Raspberry Coffee/CoffeeMakerControllerApp/project.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "dependencies": {
+    "Microsoft.ApplicationInsights": "1.0.0",
```
Comment (bfjelds) [inline Raspberry Coffee/CoffeeMakerControllerApp/project.json]: If we don't actually care about the telemetry for this sample, remove the ApplicationInsights references everywhere.

**Comment by bfjelds** [comment]
replace the pfx file here with one copied and renamed from another sample (they are all tied to MSFT as a publisher and have the same expiration date)

**Comment by trmck** [comment]
All changes have been made and committed.

**Pull request merged**